.github/workflows/build_develop.yml

```diff
@@ -34,11 +38,16 @@ jobs:
           SENTRY_URL: ${{ secrets.SENTRY_URL }}
           SENTRY_ORG: element
           SENTRY_PROJECT: riot-web
+          # We have to update the CSP until Cloudflare Pages supports proxying redirects
+          # We only deploy the latest bundles to Cloudflare Pages and use _redirects to fallback to R2 for
+          # older ones, this redirect means that 'self' is insufficient in the CSP.
+          # Once Cloudflare redirects support proxying mode we will be able to ditch this.
+          # https://developers.cloudflare.com/pages/platform/redirects/#:~:text=the%20incoming%20request-,Proxying,-No
```

Comment:
I don't think this `#:~:text=` anchor thing works?

Reply:
I think it is chrome-specific

Reply:
hrm. Given it's chrome-specific, maybe it would be good to say "see Proxying at <URL>" ?